### DISCUSSION TITLE
C310 model and other in Unreal Engine

### DISCUSSION THREAD

**Discussion by MariaALJ** [JSBSim-Team/jsbsim#861] [Q&A]
Hello, i followed what was said in discussion #826 and I used the PW125BX engine instead of the default in the C310 model but it's still not working. The aircraft is still rotating around itself at play. Are any other changes needed in order to make it work?

And I was wondering if any control model was created for the ah1s or any reference that would help understand how to make ah1s fly correctly?
Same question for the F450 quadropter 
Thank you in advance!

**Comment by jonyMarino** [comment]
@MariaALJ I think you are talking about the C310 model. I have been able to run it with waypoints and everything (not on the Unreal Plugin). Try first to run the scripts under the scripts/ folder successfully. Then you can compare it with Unreal.

**Reply by MariaALJ** [reply]
Thank you for your reply. Sure yeah my bad, I'm talking about C310. And thank you for the advice!

**Comment by gallonmate** [comment]
The C310 works with no modifications but you have to start the engine's manually in the game (I'm not sure why, seems like a bug). 
Press CTRL+ Q at same time to toggle engine starter on and off.

AH1S is an experimental flight model. Did you see this discussion on the AH1S and my comment there? https://github.com/JSBSim-Team/jsbsim/discussions/659#discussioncomment-3793664

Here are basic commands to get the AH1S flying. This turns the engine govern on to give power to the rotor. And you can control the Collective.
![image](https://user-images.githubusercontent.com/28702753/229411550-dfac96b6-f745-4381-a600-ee82e5db5a28.png)

Then as seen in my comment I linked, https://github.com/JSBSim-Team/jsbsim/discussions/659#discussioncomment-3793664 I described using these Stabilization commands, otherwise it's almost impossible to control the AH1S in a steady flight. ![image](https://user-images.githubusercontent.com/28702753/193757898-d7c75bb6-493d-4b9a-bcb0-b6e906e3a328.png)

To fly the F450, I have no idea. JonyMarino had good advice. That's what I did to fiqure out the AH1S and other aircraft, you search the git repo for test flight scripts and see what commands they use in the scripts to control the aircraft. I then use those commands in UE5 myself.
